pkg/tms/tmsUtils.go

```diff
@@ -140,6 +146,14 @@ func unmarshalServiceKey(serviceKeyJson string) (serviceKey serviceKey, err erro
 	if err != nil {
 		return
 	}
+	if len(serviceKey.Uri) == 0 {
+		if serviceKey.CALMEndpoints != nil && len(serviceKey.CALMEndpoints.API) > 0 {
+			serviceKey.Uri = serviceKey.CALMEndpoints.API + CALM_REROUTING_ENDPOINT_TO_CTMS
```

Comment:
As mentioned by @HoffmannThomas the endpoint to the cTMS service cannot be retrieved by REST request from the cALM API, so we will hardcode it here.
I chose to add it to this function as it was simplest to write unit tests for.